### ISSUE TITLE
Cursor is not animated in Hyprland

### ISSUE THREAD

**Issue by tfkhdyt** [danth/stylix#483]
I'm using [`apple-cursor`](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/data/icons/apple-cursor/default.nix#L37) in Hyprland, but the wait/loading cursor is not animated.
Is it an expected behavior when using Stylix in Hyprland? 
My stylix config:
```
cursor = {
  package = pkgs.apple-cursor;
  name = "macOS-Monterey";
  size = 24;
};
```

It happens in other cursor theme as well, for example [`bibata-cursors`](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/by-name/bi/bibata-cursors/package.nix#L52)

**Comment by danth** [comment]
This could be a limitation of Hyprland itself, or of the app you're using. I'm not aware of anything Stylix does which would prevent an animated cursor from working - we basically just pass the cursor package through unmodified and enable it:

https://github.com/danth/stylix/blob/0ef70039a6435446472182c8f8106947abfc523d/stylix/hm/cursor.nix#L10-L14

**Comment by trueNAHO** [comment]
Feel free to open an issue upstream.

**Comment by tfkhdyt** [comment]
This issue has been fixed after hyprland `0.42.0` update
